src/components/registerHeader/styles.ts

```diff
@@ -0,0 +1,46 @@
+import { Box } from '@mui/system';
+import styled from 'styled-components';
+import { Button } from '../button';
+import theme from '../../utils/theme';
+import { Typography } from '@mui/material';
+
+export const ContainerUp = styled(Box).attrs({
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  margin: '16px 40px 0 40px',
+})`
+  @media (max-width: ${theme.breakpoints.values.sm}px) {
+    margin: 16px 40px 0 40px !important;
```

Comment:
Aqui é pra ser assim:

```suggestion
    margin: 16px 16px 0 16px !important;
```

No celular tá ficando muito espaçado nas laterais do jeito que tá:
<img width="366" alt="Captura de Tela 2023-05-27 às 19 42 42" src="https://github.com/ajuda-ai-ufam/frontend/assets/26875510/d8d33365-9753-42dc-9cd9-0460c9c05f66">